cmd/root.go

```diff
@@ -765,7 +498,9 @@ func createComments(ghIssue github.Issue, jIssue jira.Issue, existing []jira.Com
 
 // updateComment compares the body of a GitHub comment with the body (minus header)
 // of the JIRA comment, and updates the JIRA comment if necessary.
-func updateComment(ghComment github.IssueComment, jComment jira.Comment, jIssue jira.Issue, ghClient github.Client, jClient jira.Client) error {
+func updateComment(config lib.Config, ghComment github.IssueComment, jComment jira.Comment, jIssue jira.Issue, ghClient github.Client, jClient jira.Client) error {
```

Comment:
same: all of these functions that are *not* directly concerned with configuring and launching the command should be outside of this file and package. this function should be in lib.